tools/depends/target/libdvdread/Makefile

```diff
@@ -16,14 +16,21 @@ else
   ifeq ($(PLATFORM),)
     PLATFORM = native
     TARBALLS_LOCATION = $(ROOT_DIR)
-    PKGCONFIGPATH = PKG_CONFIG_PATH="$(ROOT_DIR)/../libdvdcss/dvdcss-install/lib/pkgconfig"
-    EXTRA_CFLAGS += -I$(ROOT_DIR)/../libdvdcss/dvdcss-install/include
+    ifneq ($(BUILD_DVDCSS),0)
```

Comment:
this disables DVDCSS support globally for every platform that builds with depends as BUILD_DVDCSS is not defined in depends.